app/mailers/notifier.rb

```diff
@@ -67,6 +67,15 @@ def statement(args)
     send_nucore_mail args[:user].email, text("views.notifier.statement.subject", facility: @facility)
   end
 
+  def offline_cancellation_notification(reservation)
```

Comment:
Haven't looked any closer than this comment, but I would lean towards breaking out into a separate mailer class. I'd like to break up this generic Notifier mailer at some point.